warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__schedule_download_success.sql

```diff
@@ -0,0 +1,28 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__schedule_feed_guideline_index') }}
+),
+
+daily_feed_download_unzip_success AS (
+    SELECT feed_key
+      FROM {{ ref('fct_schedule_feed_downloads') }}
+     WHERE download_success
+       AND unzip_success
+),
+
+int_gtfs_quality__schedule_download_success AS (
+    SELECT
+        idx.date,
+        idx.feed_key,
+        {{ static_feed_downloaded_successfully() }} AS check,
+        {{ compliance() }} AS feature,
+        CASE
+            WHEN LOGICAL_AND(d.feed_key IS NOT null) THEN "PASS"
+            ELSE "FAIL"
+        END AS status
```

Comment:
Also one code cleanliness note, you could keep all these status columns as booleans and then just convert to strings in the final fact table

It might make some downstream aggregations easier, too, if you keep the booleans around